### ISSUE TITLE
Selection of screen reader at app startup

### ISSUE THREAD

**Issue by justdroid** [saurabh2590/EasyAccess#16]
BUG
At launch of the app, it prompts to select the screen reader / text to speech engine to be used.

DETAILS
A few users have reported that in cases where more than one text-to-speech (TTS) engine is installed on user's phone, the app asks for selection of that TTS every single time the app is started. We have not been able to verify this as we don't have multiple screen readers installed (we only have the native TalkBack screen reader to test) but should be tested on phone that has two screen readers installed.

LIKELY MODULE
SplashActivity.java

**Comment by manishoswal** [comment]
This is a default android feature. If we have more than one app that can perform the same intent , android shows the app chooser dialog. If the user selects the **"Always"** option , it will not show the dialog till we change the preference from settings screen.. 
We cannot modify this behavior

.
